tests/includes/activity/class-test-activity-dispatcher.php

```diff
@@ -5,12 +5,16 @@
  * @package Activitypub
  */
 
+namespace Activitypub\Tests\Activity;
```

Comment:
this is (regardless of the name) in the root includes folder. Just to keep it consistent, until it will be deleted by the inbox PR ;)